mongodb.md

```diff
@@ -76,10 +75,10 @@ Within this file, you can add a `mongodb` connection using the `mongodb` driver:
 
 Once your configuration is complete, you can use the `mongodb` connection in your application:
 
-- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Using raw SQL queries is not supported obviously, as MongoDB is a NoSQL database.
+- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Raw SQL queries is not supported but advanced MongoDB queries can be used.
```

Comment:
```suggestion
- [Using Eloquent](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/), models can be stored in MongoDB collections. In addition to the standard Eloquent features, the Laravel MongoDB package provides additional features such as embedded relationships. Raw SQL queries are not supported but advanced MongoDB queries can be used.
```

Fixing subject/verb agreement.

"advanced MongoDB queries can be used" sounds a bit vague. Would it be clearer to say:

> The package also provides direct access to the MongoDB driver, which can be used to execute operations such as raw queries and aggregation pipelines. This is analogous to Eloquent's API for executing raw SQL queries (not supported by MongoDB).

Maybe just remove the second sentence, as I think any mention of "raw SQL queries" is likely to cause more confusion than anything.